ansible/roles/gcp-cloud-storage/tasks/gcloud-auth.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Copy service account key file
+  copy: 
+    content: "{{ gcp_storage_key_file }}" 
+    dest: /tmp/gcp_storage_key_file.json
```

Comment:
Can we prefix this with some unique value that is specific for this run only only? For instance, if two jobs run at the same time on the same VM, chances are they will be overwritten. Maybe an epoch timestamp can give us certain level uniqueness.